.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,4 +30,6 @@ Fixes #?
 
 - [ ] Documentation 
 
+- [ ] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc) properly
```

Comment:
The sentence sense ungentle to me. This can be done through a polite statement something like  
```
Read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)
```

Reply:
@Team Need your attention because this change going to affect all while creating a pr (Specifically test contribution)